conf/initd/webvirtmgr-novnc-redhat

```diff
@@ -32,6 +37,7 @@ start() {
     [ -x $exec ] || exit 5
 
     echo -n $"Starting $prog: "
+    export PYTHONPATH=$PYTHONPATH:/usr/lib/python2.6/site-packages/Django-1.5.6-py2.6.egg
```

Comment:
You're welcome! :)
The RPM is designed to work on cleaned RedHat-like system (RHEL, CentOS) installations. If you add EPEL repo, you have all tools necessary to work with ease.